.circleci/config.yml

```diff
@@ -516,11 +516,17 @@ workflows:
             - build
           context: slack-cli-release
           release_ref: << pipeline.git.branch >>
+          filters:
+            branches:
+              ignore: ^pull\/.*
```

Comment:
🔗 https://circleci.com/docs/workflows/#using-filters-in-your-workflows

This guards against branches matching "pull/123/head" which is the format used to start tests from remotes at this time.

Reply:
Might be worth adding a comment to note that this pattern intentionally guards against matching "pull/123/head"